src/test/java/com/adobe/cq/commerce/graphql/client/impl/ConcurrencyTest.java

```diff
@@ -122,23 +92,19 @@ public void testConcurrencyWithErrors() throws Exception {
         // If the error messages are not consumed by the client, this test will time out
         // because all threads from the connection pool will stay stuck
 
-        HttpResponse error = new HttpResponse()
-            .withStatusCode(INTERNAL_SERVER_ERROR_500.code())
+        HttpResponse error = new HttpResponse().withStatusCode(INTERNAL_SERVER_ERROR_500.code())
```

Comment:
I'm being picky, but you should avoid reformatting code you don't change, it unnecessarily "pollute" the PR with unrelated changes.